### PR TITLE
6.10.x-release with COPPA option

### PR DESCRIPTION
**Pull request by jiancchen**

### CODE REVIEW

**Review by jeffreychang-vungle — APPROVED** [review]

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleInitializer.java [hunk 1]
```diff
@@ -120,7 +120,6 @@ public void updateCoppaStatus(int configuration) {
         Vungle.updateUserCoppaStatus(true);
         break;
       case RequestConfiguration.TAG_FOR_CHILD_DIRECTED_TREATMENT_FALSE :
-      case RequestConfiguration.TAG_FOR_CHILD_DIRECTED_TREATMENT_UNSPECIFIED :
         Vungle.updateUserCoppaStatus(false);
         break;
       default:
```
Comment (palrahul) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleInitializer.java:125]: @jiancchen 
recommend to update this

`
 public void updateCoppaStatus(int configuration) {
    switch (configuration) {
      case RequestConfiguration.TAG_FOR_CHILD_DIRECTED_TREATMENT_TRUE :
        Vungle.updateUserCoppaStatus(true);
        break;
      case RequestConfiguration.TAG_FOR_CHILD_DIRECTED_TREATMENT_FALSE :
        Vungle.updateUserCoppaStatus(false);
        break;
      case RequestConfiguration.TAG_FOR_CHILD_DIRECTED_TREATMENT_UNSPECIFIED:  
      default:
        //Vungle's SDK only supports setting COPPA_ENABLED or COPPA_NOT_ENABLED status
        //For Unspecified case Publishers will have to enable COPPA  on Vungle's dashboard
        //and then Vungle will accordingly treat any user whose COPPA status is not known 
        //according to the compliance requirements
        break;
    }
  }

`

Comment (jiancchen) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleInitializer.java:125]: @palrahul sure this adds more clarity

**Review by palrahul — APPROVED** [review]
lgtm

**Comment by HuFangshuai** [comment]
Close this PR due to the functionality has been released in the upstream repo.

**Pull request closed without merging**